bus/bus.go

```diff
@@ -82,6 +85,15 @@ type (
 		TriggerUpdate()
 	}
 
+	Syncer interface {
```

Comment:
I think you removed it somewhere but I moved it to `bus.go`. I add an `AlertManager` in https://github.com/SiaFoundation/renterd/pull/1435 which makes it so all `bus` dependencies are properly interfaced again.